test/index.spec.ts

```diff
@@ -135,17 +133,17 @@ describe.each(Object.keys(configuredRules))(
      * vitest fails if there aren't any tests in a suite,
      * so filter out rules that don't have fixtures or are turned off
      */
-    const rulesWithFixtures = Object.keys(
-      configuredRules[ruleSet].rules,
```

Comment:
changes look bigger than they are, in reality just removed `.rules` as the `configuredRules` object is a bit less nested now